### ISSUE TITLE
Replicated MongoDB extended test fails

### ISSUE THREAD

**Issue by mfojtik** [openshift/origin#8753]
```console
  creating from a template
  /data/src/github.com/openshift/origin/test/extended/images/mongodb_replica.go:68
    should process and create the "https://raw.githubusercontent.com/openshift/mongodb/master/2.4/examples/replica/mongodb-clustered.json" template [It]
    /data/src/github.com/openshift/origin/test/extended/images/mongodb_replica.go:67

    Expected success, but got an error:
        <*errors.errorString | 0xc2085d20f0>: {
            s: "timed out waiting for deployment \"mongodb\" after 15m0s",
        }

    /data/src/github.com/openshift/origin/test/extended/images/mongodb_replica.go:39
```

@php-coder found in logs that the problem might be related to the admin user creation.

The flake can be seen here: https://ci.openshift.redhat.com/jenkins/job/test_pr_origin_extended/152/

**Comment by php-coder** [comment]
>@php-coder found in logs that the problem might be related to the admin user creation.

Because CI logs may disappear in the future, here is the related piece of the log from `container-mongodb-1-hook-post-lifecycle.log`:
```
=> Creating MongoDB users ...
MongoDB shell version: 2.4.9
connecting to: localhost:27017/admin
{
	"user" : "admin",
	"pwd" : "87a80ce9b2643ffda03f58c09032ac23",
	"roles" : [
		"dbAdminAnyDatabase",
		"userAdminAnyDatabase",
		"readWriteAnyDatabase",
		"clusterAdmin"
	],
	"_id" : ObjectId("572afbc67cc309b658c54de5")
}
Thu May  5 07:52:38.479 couldn't add user: not authorized for insert on admin.system.users at src/mongo/shell/db.js:128
=> Failed to create MongoDB admin user.
```

**Comment by php-coder** [comment]
Locally if fails for me with the a bit different error:
```
Thu May  5 12:47:45.489 [conn12] command admin.$cmd command: { getlasterror: 1.0, w: "majority", wtimeout: 30000.0 } ntoreturn:1 keyUpdates:0  reslen:242 6739ms
MongoDB shell version: 2.4.9
connecting to: localhost:27017/admin
{
        "user" : "dT4",
        "pwd" : "e071576e8972d9b1e3c772017ce89c09",
        "roles" : [
                "readWrite"
        ],
        "_id" : ObjectId("572b40f1f071ad729683feda")
}
Thu May  5 12:47:45.517 couldn't add user: not authorized for insert on sampledb.system.users at src/mongo/shell/db.js:128
=> Failed to create MongoDB user: dT4
```

**Comment by php-coder** [comment]
It's not a flake but just a bug in the mongodb image.

**Comment by bparees** [comment]
@php-coder should this be closed since #154 merged?

**Comment by php-coder** [comment]
Yes, I believe that we can close this issue.
